src/main/java/school/lemon/changerequest/java/extendedinteger/ExtendedInteger.java

```diff
@@ -16,13 +16,13 @@ public static boolean isEven(int value) {
     }
 
     public static boolean isOdd(int value) {
-        return (value % 2 != 0);
+        return !((value & 1) == 0);
```

Comment:
you've got equality and negation but there is really no need for them. there are 2 possible good ways to implement this method:
```java
public static boolean isOdd(int value){
  return (value&1) == 1;
}
```
And:
```java
public static boolean isOdd(int value){
  return !isEven(value);
}
```